autoinstrumentation/nginx/Dockerfile

```diff
@@ -0,0 +1,25 @@
+
+############################
+# STEP 1 download the webserver agent 
+############################
+FROM alpine:latest as agent
+
+ARG version
+
+RUN mkdir /opt/opentelemetry
+WORKDIR /opt/opentelemetry
+
+RUN wget https://github.com/open-telemetry/opentelemetry-cpp-contrib/releases/download/webserver%2Fv$version/opentelemetry-webserver-sdk-x64-linux.tgz
+RUN mkdir agent
+RUN tar -xvf opentelemetry-webserver-sdk-x64-linux.tgz -C agent
+
+############################
+# STEP 2 build streamlined image
+############################
+FROM alpine:latest
```

Comment:
does this need to be alpine? Could we change it to `from scratch`?

Reply:
interesting idea - i think it has to have the busybox at least - let me give it a try.

Reply:
if I build an image with the instrumentation libraries and a go-based utility which does all the instrumentation logic, which runs without any dependencies in `from scratch` built image - would that be ok?